bin/express-cli.js

```diff
@@ -308,6 +319,11 @@ function createApplication (name, path) {
       case 'vash':
         pkg.dependencies['vash'] = '~0.12.2'
         break
+      case 'jsx':
+        pkg.dependencies['react'] = '^16.2.0'
```

Comment:
From our current practices, these should at most be `~` or just pinned; no `^` dependencies :)